CHANGES

```diff
@@ -1,5 +1,8 @@
 This file describes changes to pypersist since the initial release.
 
+1.1 (2020-08-26)
```

Comment:
This should be 1.0.1 and the date should be something like "2020-??-??" for now.  I hope this will be in a release soon, but certainly not today! :)